frontend/app/templates/components/team-member-configure.hbs

```diff
@@ -0,0 +1,80 @@
+<div class="modal fade" tabindex="-1" role="dialog" aria-labelledby="teamConfigureModalLabel" aria-hidden="true" {{did-insert this.setupModal this}}>
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h3 class="modal-title" id="teamConfigureModalLabel">{{t "team.editMembersAndApiUsers"}}</h3>
+        <button type="button" class="close" data-dismiss="modal" aria-label={{t "close"}}>
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <ul class="nav nav-tabs" role="tablist">
+          <li class="nav-item active">
+            <a class="nav-link" id="members-tab" data-toggle="tab" href="#members" role="tab" aria-controls="members" aria-selected="true">{{t "team.members"}}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" id="api-users-tab" data-toggle="tab" href="#api-users" role="tab" aria-controls="api-users" aria-selected="false">{{t "team.apiUsers"}}</a>
+          </li>
+        </ul>
+        <div class="tab-content">
+          <div class="tab-pane active" id="members" role="tabpanel" aria-labelledby="members-tab">
+            <div class="columns members">
+              <div class="row justify-content-end">
+                <div class="col-sm-4 float-right">
+                  <PowerSelectTypeahead @renderInPlace="true" @options={{this.candidates}} @search={{this.search}} @onChange={{this.addMember}} as |candidate|>
+                    {{candidate.label}}
+                  </PowerSelectTypeahead>
```

Comment:
Not yet properly styled. Not sure how to add this:
https://github.com/cibernox/ember-power-select-typeahead#styles